src/config.sh

```diff
@@ -206,6 +214,12 @@ function parse_config_options()
   # Default values
   options_values['SCOPE']='local'
   options_values['PARAMETERS']=''
+  options_values['IS_SHOW_CONFIGURATIONS']=false
+
+  # 'kw config' should list all configurations
+  if [[ "$#" == 0 ]]; then
```

Comment:
I think you want to move this condition to the `case` below.

Reply:
That was my intention, but I couldn't find a way to make it work. The problem is that in the `case` below, I don't think there is a way to recuperate the info of "no arguments passed". I based my approach in [this part](https://github.com/kworkflow/kworkflow/blob/unstable/src/kernel_config_manager.sh#L31) of the `kw k` implementation.